website/docs/r/managed_policies.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+subcategory: "Policies"
+layout: "lacework"
+page_title: "Lacework: lacework_managed_policies"
+description: |-
+  Manage Lacework Defined Policies
+---
+
+# lacework\_managed\_policies
+
+Use this resource to update the "enabled" and the "severity" property for Lacework defined policies.
```

Comment:
@lacework-aaronscheiber Do you think you can help us reviewing this customer-facing doc?